libs/log.c

```diff
@@ -57,6 +58,12 @@ log_open(void)
 		expanded_path = expand_path(path);
 
 		path = fxstrdup(expanded_path);
+		if (expanded_path[0] == '/')
+		{
+				path = fxstrdup(expanded_path);
```

Comment:
The logic here is incorrect (not to mention this is now causing a memory leak).

We set `path` to `expanded` path, and then set `path` to `expanded_path` again if `expanded_path` begins with a `/`.

This isn't right.  Can someone please fix this logic?